### PR TITLE
mizu-cli: Add version 33.1

### PR DESCRIPTION
**Pull request by ogerardin**
Added manifest for mizu-cli

- [X] I have read the [Contributing Guide](https://github.com/ScoopInstaller/.github/blob/main/.github/CONTRIBUTING.md).

### CODE REVIEW

**Comment by ogerardin** [comment]
The test failures don't seem related to this PR... Is there something wrong with the automated tests ?
`Unable to resolve package source 'https://www.powershellgallery.com/api/v2'.`

**Comment by rashil2000** [comment]
Add checkver and autoupdate

File: bucket/mizu-cli.json [hunk 1]
```diff
@@ -0,0 +1,13 @@
+{
+    "version": "33.1",
+    "description": "The API Traffic Viewer for Kubernetes",
+    "homepage": "https://getmizu.io/docs/mizu/mizu-cli",
+    "license": "Apache-2.0",
```
Comment (issaclin32) [inline bucket/mizu-cli.json:5]:
```suggestion
    "license": "Apache-2.0",
```
The trailing whitespace is causing CI to fail.

**Comment by issaclin32** [comment]
> The test failures don't seem related to this PR... Is there something wrong with the automated tests ? `Unable to resolve package source 'https://www.powershellgallery.com/api/v2'.`

The problem disappeared after re-running the job. Probably a Github server issue?

File: bucket/mizu-cli.json [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+    "description": "The API Traffic Viewer for Kubernetes",
+    "homepage": "https://getmizu.io/docs/mizu/mizu-cli",
+    "license": "Apache-2.0",    
+    "architecture": {
+        "64bit": {
+            "url": "https://github.com/up9inc/mizu/releases/download/33.1/mizu.exe",
+            "hash": "1da9381b9267016454304b5dbc33dfd6740e8d2d30c33721ea3215071bac271c"
+        }
+    },
+    "bin": "mizu.exe"
```
Comment (issaclin32) [inline bucket/mizu-cli.json:12]:
```suggestion
    "bin": "mizu.exe",
    "checkver": {
        "github": "https://github.com/up9inc/mizu"
    },
    "autoupdate": {
        "architecture": {
            "64bit": {
                "url": "https://github.com/up9inc/mizu/releases/download/$version/mizu.exe",
                "hash": {
                    "url": "$baseurl/mizu_windows_amd64.sha256"
                }
            }
        }
    }
```

**Comment by ogerardin** [comment]
@issaclin32 Thanks for adding the checkver!

File: bucket/mizu-cli.json [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+    "checkver": {
+        "github": "https://github.com/up9inc/mizu"
+    },
+    "autoupdate": {
+        "architecture": {
+            "64bit": {
+                "url": "https://github.com/up9inc/mizu/releases/download/$version/mizu.exe",
+                "hash": {
+                    "url": "$baseurl/mizu_windows_amd64.sha256"
+                }
```
Comment (tech189) [inline bucket/mizu-cli.json:22]:
```suggestion
                    "url": "$baseurl/mizu_windows_amd64.sha256",
                    "regex": "$sha256"
                }
```
Fixes autoupdate hash extraction

**Review by issaclin32 — APPROVED** [review]

**Comment by ogerardin** [comment]
Thanks everyone!

**Pull request merged**